src/net/quality.rs

```diff
@@ -119,6 +119,6 @@ mod test {
         let smoothed_rtt = network_quality.smooth_out_rtt(300);
 
         // 300ms has exceeded 50ms over the max allowed rtt. So we check if or smoothing factor is now 10% from 50.
-        assert_eq!(smoothed_rtt, 5.0);
+        assert_eq!((smoothed_rtt - 5.0f32).abs() < std::f32::EPSILON, true);
```

Comment:
what is this doing?

Reply:
Clippy shows an error if I try to compare two floats for equality, and it should.
So what I do, is check if their absolute difference is smaller than epsilon.
It only returns true if smoothed_rtt is between ~4.99999... and ~5.00000001...